content/news/2020-12-07-contest-results.md

```diff
@@ -0,0 +1,63 @@
++++
+type = "news"
+title = "System Sounds Contest Wrap-up"
+author = "bitigchi"
+date = "2020-12-07 22:00:00+03:00"
+tags = ["sound", "contest"]
++++
+
+Contest for Haiku System Sounds is finally over; and ladies and gentlemen, we have a winner!
+
+Before going into details, lots of thanks to all participants and voters are in order. Thanks to you, Haiku now has a cohesive high-quality sound set, that will hopefully make using Haiku much more pleasant and accessible.
+
+Without further ado, the winner is **Garrett Kabler**, with the sound theme "**burg's-sparkly-tones**"!
+
+Garrett's entry won the contest by a landslide, acquiring a whopping 249 points, whilst the runner up followed up with 50 points. This is very impressive, and gives an overall idea about the quality of the Garrett's sound set.
+
+Top three are listed below:
+
+1. Garrett Kabler - [burg's-sparkly-tones](/files/sound-contest/garrett_kabler_burgs-sparkly-tones.zip)
+2. Matthew Harazim - [Komorebi](/files/sound-contest/bearlyMatt-Komorebi.zip)
+3. Maksim Malerson - [Basics](/files/sound-contest/maksim_malerson_Basics.zip)
+
+For the details about the voting and the final scores, see the "Statistics" section below.
+
+## Statistics
+
+For the evaluation, entries were ranked from one to five, each of them corresponding to below points:
+
+1. 1st place: 5 points
+2. 2nd place: 4 points
+3. 3rd place: 3 points
+4. 4th place: 2 points
+5. 5th place: 1 point
+
+According to votes, final score were calculated as below:
+
+| **Theme name**                    |**Points**|
+|:------n----------------------------------|--:|
+|**1.** burg's-sparkly-tones               |249|
+|**2.** Komorebi                           | 50|
+|**3.** Basics                             | 43|
+|**4.** Zen                                | 42|
+|**5.** Clack                              | 37|
+|**6.** Yanqim                             | 25|
+|**7.** Leafy                              | 23|
+|**8.** Forest Dreams                      | 23|
+|**9.** Low Modularian                     | 18|
+|**10.** Zen Peace                         | 16|
+|**11.** Meditation Time                   | 15|
+|**12.** Bassling                          | 15|
+|**13.** Straight Outta Cronton            | 13|
+|**14.** Nature                            | 12|
+|**15.** Supposedly Not Too Annoying Theme | 10|
+|**16.** The Poet Operating System Theme   |  7|
+|**17.** Daniel Diaz                       |  6|
+
+You can download the votes' breakdown table from [this link](/files/sound-contest/sound_contest_cotes.ods) (.ods file, 20 KiB).
+
+## What next?
+
+Winner entry will be used as the default sound theme in Haiku R1. Expect to see this theme in action in the upcoming beta releases. You can still use any sound theme you please via setting them up manually in "Sounds" preferences. It will also be possible to install and use the other entries via HaikuDepot, once the underlying foundation for this feature is implemented.
```

Comment:
Let's not promise this for the next beta. "Sounds" prefs need to support themes, or at the least muting sounds. I'm pretty sure we'll decide to have all sounds (besides startup and beep) muted by default.

```suggestion
The winner entry will be used as the default sound theme in Haiku R1 once proper theming is implemented in the "Sounds" preferences. You can still use any sound theme you please (or none) via setting them up manually in the "Sounds" preferences. It will also be possible to install and use the other entries via HaikuDepot, once the underlying foundation for this feature is implemented.
```

Reply:
I think we can already use the winner theme by default, no? Profiling is only needed if the other sound themes were to be added.

Reply:
Yes. But I would hate to have all those sound events active by default and having to clear all of them manually.

Reply:
It makes sense, but the last time I checked, the improvements ticket which included the mock-up for profiling was not under R1 milestone. If no one implements profiling, the default theme will probably be included anyway I guess.

Reply:
You mean, included, but not pre-configured in the Sounds prefs?

Reply:
I think they would be pre-configured... By BeOS' logic, someone has to manually replace the sounds anyway if they want to customise them.

Reply:
As I said, I think most people would hate to boot into a newly installed Haiku with every mouse click, key type and window action beep-beep-booping until I manually set those dozen events manually to <none>.
As we don't know how it will be done eventually, we should phrase it more fuzzy. How about:

```suggestion
The winning sound theme will be included in Haiku R1. You can still use any sound theme you please via setting them up manually in "Sounds" preferences or install and use the other entries via HaikuDepot, once the underlying foundation for this feature is implemented.
```

Reply:
Updated. I don't know about profiling, but on/off checkboxes should definitely be implemented before R1, and most of them should be off by default.

Reply:
+1

Reply:
In case it's not implemented, we can just include the prominent ones, like notifications, startup etc.